src/lib/config.ts

```diff
@@ -46,6 +46,18 @@ export const requiredVar = (varName: string): string => {
 	return s;
 };
 
+export function optionalVar(varName: string, defaultValue: string): string;
+export function optionalVar(
+	varName: string,
+	defaultValue?: string,
+): string | undefined;
```

Comment:
Unfortunately I need to repeat this (even though it's exactly the same as the implementation) since only the overload signature definitions are used while consuming the function.
The implementation's param types are not used anywhere else except the func implementation itself.